cheri_compressed_cap_common.h

```diff
@@ -83,6 +92,8 @@ struct _cc_N(cap) {
     uint8_t cr_flags;    /* Flags */
```

Comment:
Do we still need cr_ebt now that there is cached_pesbt?

Reply:
We don't need it, and I _think_ we don't want it. We would have to refactor bounds setting and tests, which still currently work in terms of ebt, and then construct pesbt correctly.

Reply:
I think it would be better if we could remove the duplicate information. Otherwise we might accidentally update the wrong value.

Reply:
Now gone